recipes/coiled-runtime/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "coiled-runtime" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/coiled/coiled-runtime/dist/conda/noarch/{{ version }}/coiled-runtime-{{ version }}.tar.gz
+  sha256: 43c98fb629bc2b877a3ad47ae41881fb14bcaf3dd587035d7bf8649cfdfe4413
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+
+  run:
+    - python >=3.8
+    - pip
+    - coiled
+    - nodejs ==17.8.0
+    - nb_conda_kernels ==2.3.1
+    - numpy ==1.21.5
+    - pandas ==1.4.1
+    - dask ==2022.1.0
+    - distributed ==2022.1.0
+    - fsspec ==2022.3.0
+    - s3fs ==2022.3.0
+    - gcsfs ==2022.3.0
+    - pyarrow ==7.0.0
+    - python-snappy ==0.6.0
+    - jupyterlab ==3.3.2
+    - dask-labextension ==5.2.0
+    - lz4 ==4.0.0
+    - ipywidgets ==7.7.0
+    - numba ==0.55.1
+    - scikit-learn ==1.0.2
+    - python-graphviz ==0.19.1
+    - click <=8.0.4
```

Comment:
Just checking, is it intentional that this `click` version constraint is `<=` instead of `==` like most of the other packages?

Reply:
It was intentional, but its a fair point.  Should probably be hard pinned as well.